flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/SqlNodeToOperationConversion.java

```diff
@@ -324,6 +338,8 @@ private static Optional<Operation> convertValidatedSqlNode(
             return Optional.of(converter.convertShowCreateView((SqlShowCreateView) validated));
         } else if (validated instanceof SqlRichExplain) {
             return Optional.of(converter.convertRichExplain((SqlRichExplain) validated));
+        } else if (validated instanceof SqlRichDescribeModel) {
+            return Optional.of(converter.convertDescribeModel((SqlRichDescribeModel) validated));
```

Comment:
Is there anything that blocks from moving it to SqlNodeConverters instead of putting here?